### PR TITLE
Remove apiextensions dependency

### PR DESCRIPTION
**Pull request by ubergesundheit**
After #2 was not working correct, the changes in here are working as intented. After merge into master, I'll create a release 0.2.0 from this after merge

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -8,32 +8,42 @@ import (
...
-	err := yaml.Unmarshal(document, &release)
+func handleDocument(provider, annotationKey string, document []byte) {
+	var releaseObject map[string]interface{}
+	err := yaml.UnmarshalStrict(document, &releaseObject)
 	if err != nil {
 		fmt.Println(err)
-		os.Exit(4)
+		os.Exit(1)
 	}
+	release := unstructured.Unstructured{Object: releaseObject}
```
Comment (tfussell) [inline main.go:22]: I forgot about the nested `Object`. Makes sense now though.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Remove dependency on giantswarm apiextensions
```
Comment (tfussell) [inline CHANGELOG.md]:
```suggestion
### Removed

- Remove dependency on giantswarm/apiextensions.
```

**Review by tfussell — APPROVED** [review]
LGTM if tested.

**Comment by ubergesundheit** [comment]
Tested here: https://app.circleci.com/pipelines/github/giantswarm/releases/2383/workflows/4a454998-5b32-41e9-b262-86853b8180bf/jobs/8806

**Pull request merged**